src/transport_performance/gtfs/multi_validation.py

```diff
@@ -379,12 +379,21 @@ def filter_to_bbox(
             self._raise_empty_feed_error(bbox)
         return None
 
+    def _summary_col_sorter(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
No unit tests for this, would it be worth adding them?

Reply:
Going to say no on this as it is an internal function. It is also tested through `test__summarise_core()`  through dataframe equality checks, including column ordering.